site/docs/ch/flink/flink-ddl.md

```diff
@@ -116,3 +116,53 @@ SHOW DATABASES;
 ```sql
 SHOW TABLES;
 ```
+
+## Supported Types
+
+### Mixed Hive Data Types
+
+| Flink Data Type    | Hive Data Type    |
+|-----|-----|
+| STRING    | CHAR(p)    |
+| STRING    | VARCHAR(p)    |
+| STRING    | STRING    |
+| BOOLEAN    | BOOLEAN    |
+| INT    | TINYINT    |
+| INT    | SMALLINT    |
+| INT    | INT    |
+| BIGINT    | BIGINT    |
+| FLOAT    | FLOAT     |
+| DOUBLE    | DOUBLE    |
+| DECIAML(p, s)    | DECIAML(p, s)    |
+| DATE    | DATE    |
+| TIMESTAMP(6)    | TIMESTAMP    |
+| VARBINARY    | BYNARY    |
+| ARRAY<T>     | ARRAY<T>     |
+| MAP<K, V>    | MAP<K, V>     |
+| ROW    | STRUCT       |
+
+
+### Mixed Iceberg Data Types
```

Comment:
I don't think Mixed-Iceberg format table has type like `CHAR(p)`、`VACHAR(p)`、`TINYINT`、`SMALLINT`、`FLOAT`、`DOUBLE`.

Mixed-Iceberg format table has the same types as the Iceberg format table.
Maybe we should complete this description refer:https://iceberg.apache.org/docs/latest/flink/#type-conversion

Reply:
Do you recommend adding Flink Types convert to Iceberg Types, and Iceberg Types convert to Flink Types two descriptions in the Mixed Iceberg Data Types section?